jbi/services.py

```diff
@@ -6,13 +6,13 @@
 from typing import TYPE_CHECKING
 
 import backoff
-import bugzilla as rh_bugzilla
+import requests
```

Comment:
I notice that we don't have `requests` in `pyproject.toml`. Should we add it?

Reply:
Yes!